crud/crud.py

```diff
@@ -152,7 +152,7 @@ async def git_verify(self, ctx, filepath):
                                       email or "famiel@tsubakibot.com")
             commiter = pygit2.Signature("Famiel", "famiel@tsubakibot.com")
             parent, ref = repo.resolve_refish(refish=repo.head.name)
-            repo.create_commit(ref.name, author, commiter, "Updating JSON", tree, [parent.oid])
+            repo.create_commit(ref.name, author, commiter, "Updating "+filepath, tree, [parent.oid])
```

Comment:
Does the arg for create_commit need to be sanitized for special characters?

Reply:
what specifically are you worried about? this is only available to people with super elevated permissions so we don't have to worry about anything malicious, but we could add some validation, if you want to send a PR

Reply:
I'm not worried about security here. I'm worried about formatting. `Updating __init__.py` coming out as "Updating __init__.py" can be obnoxious.

Reply:
oh, that's not what itll look like. and these are predetermined json files anyway